conjure-java-core/src/integrationInput/java/com/palantir/product/BearerTokenAliasExample.java

```diff
@@ -45,4 +55,55 @@ public static BearerTokenAliasExample valueOf(String value) {
     public static BearerTokenAliasExample of(BearerToken value) {
         return new BearerTokenAliasExample(value);
     }
+
+    public static final class BearerTokenAliasExampleConverter
+            implements Converter<BearerToken, BearerTokenAliasExample> {
+        @Override
+        public BearerTokenAliasExample convert(BearerToken value) {
+            return of(value);
+        }
+
+        @Override
+        public JavaType getInputType(TypeFactory typeFactory) {
+            return typeFactory.constructType(new TypeReference<BearerToken>() {});
+        }
+
+        @Override
+        public JavaType getOutputType(TypeFactory typeFactory) {
+            return typeFactory.constructType(BearerTokenAliasExample.class);
+        }
+    }
+
+    public static final class BearerTokenAliasExampleDeserializer
```

Comment:
We could probably have an `AliasDeserializer` in the runtime lib that all aliases point to. It'd have to reflectively do the conversion by finding the `of` constructor, and requires a change to conjure-java-runtime first, but makes this file more readable.